charts/registry-scanner/templates/secret.yaml

```diff
@@ -10,4 +10,15 @@ data:
   secureAPIToken: {{ required "A valid .Values.config.secureAPIToken is required" .Values.config.secureAPIToken | b64enc | quote }}
   registryUser: {{ required "A valid .Values.config.registryUser is required" .Values.config.registryUser | b64enc | quote }}
   registryPassword: {{ required "A valid .Values.config.registryPassword is required" .Values.config.registryPassword | b64enc | quote }}
+{{- end }}
+---
+{{- range $index, $cert := .Values.ssl.ca.certs }} 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "registry-scanner.fullname" $ }}-ca-{{$index}}
+  labels:
+{{ include "registry-scanner.labels" $ | indent 4 }}
+data:
+  root_ca_file_{{$index}}.crt: {{ $cert | b64enc | quote }}
```

Comment:
I'm ok with this approach, but in other charts we also split into different files (i.e. a "secret.yaml" for the default mandatory ones, and a secret-custom-ca.yaml for the custom CAs). I leave the decision to you :)

Reply:
Well. I copied this approach from the admission controller one... ;)